### DISCUSSION TITLE
Guidelines for podcast file management and workflows

### DISCUSSION THREAD

**Discussion by maphew** [thechangelog/changelog.com#453] [General]
I have an archive of a few hundred audio files digitized from tape which I'm slowly turning into podcasts. I'm struggling with how to organize the work in a way that future me is less likely to be cursing past me. I kind of like how the Changelog Podcast repositories are organized, show notes here, transcripts there, etc. Are there particular reasons they are separate repositories instead of folders within a single repo? My default position would be to have something like `Episode_Name/transcript` and `Episode_Name/show_notes` and I'm wondering if there are some tool conditions or workflows that make that too much friction. Also the media files aren't represented. Is there reasoning behind that? (Perhaps because they're inappropriate fit for Github or something. ie. if you were self-hosting they'd be together?)

On another tack, maybe better as a separate thread but let's start here anyway: The audio files need editing, but there needs to be history so potentially someone can go back and do better job of normalizing volume and removing noise from segment X. So, source code management for audio files. Is anyone aware of tools aligned with this purpose?

**Comment by jerodsanto** [comment]
> I kind of like how the Changelog Podcast repositories are organized, show notes here, transcripts there, etc. Are there particular reasons they are separate repositories instead of folders within a single repo?

We did this in stages. Transcripts came first and it was our desire to have them open source and collab-able that drove our storage on GitHub. Once that was a success we added show notes, which we manage 99% inside our admin and sync back and forth to GitHub for the same reason: so people can submit fixes/improvements/etc

I hope that helps explain why they're separate repos and how we think about them.

> Also the media files aren't represented. Is there reasoning behind that?

The mp3s are 50MB+ in size. Not great for git. GitHub does have large file storage support, but we store them on S3 because that makes more sense for us. We need to serve them publicly at the end of the day, so Fastly + S3 makes that pretty easy.

> So, source code management for audio files. Is anyone aware of tools aligned with this purpose?

Our editing/production workflow is completely separate from our mp3 hosting and publishing workflow. We use Adobe Audition + Dropbox to accomplish our production needs, but there is no "source code management" like they way you may be thinking about it. If you find some great tools in this space, I'd love to hear about them. Right now our "version control" is old school "have a different name for each phase" style 😭 

<img width="244" alt="Screenshot 2023-03-11 at 12 01 47 PM" src="https://user-images.githubusercontent.com/8212/224504444-bcd510b3-38e5-46e5-a4be-796267710ec8.png">

Hope this helps you on your journey!

**Reply by maphew** [reply]
Thank you, that helps. I will indeed shout out if/when I find media tools with a versioning/auditing bent!
